src/main/webapp/forms/array.addselect.jsp

```diff
@@ -51,26 +51,28 @@
             i++;
         }
 
-        //Create new drop down list
-        out.println("<tr>");
-        out.println("<td>");
-        out.println("<select name=\"" + datenfeld + "[" + i + "]\" id=\"" + datenfeld + "[" + i + "]\">");
+        //Create new drop down list when Backend/Admin, not Guest
+        if ((!isReadOnly)) {
```

Comment:
Looks strange because:
- The beginning of the <select> is in the if branch, but the end </select> isn't
- To me it looks like there is no value shown if isReadOnly == true